pkgs/development/python-modules/pymodes/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchPypi, buildPythonPackage, numpy, pyzmq }:
+
+buildPythonPackage rec {
+  pname = "pyModeS";
+  version = "2.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1r85scs99c87mqkjalw1wxzscnibyyvzdgyj6q8l0f8r5f2864c2";
+  };
+
+  propagatedBuildInputs = [ numpy pyzmq ];
+
+  # No tests included
+  doCheck = false;
```

Comment:
The upstream repo contains a folder [`tests`](https://github.com/junzis/pyModeS/tree/master/tests). Please use `fetchFromGitHub` to get the complete source  and run the tests.

Reply:
Done.